skimage/measure/__init__.py

```diff
@@ -7,7 +7,7 @@
 from ._pnpoly import points_in_poly, grid_points_in_poly
 from ._moments import moments, moments_central, moments_normalized, moments_hu
 from .profile import profile_line
-from .fit import LineModel, CircleModel, EllipseModel, ransac
+from .fit import LineModel, LineModel3D, CircleModel, EllipseModel, ransac
```

Comment:
I tend to side with Kevin's original intuition that it will be confusing to
return models with different APIs, depending on the situation.

I'd suggest:

- Deprecate LineModel over two releases
- Add LineModelND right now
- A release or two after LineModel has been deprecated, rename add an alias
LineModel = LineModelND

It may also be nice to have functionality for projecting points onto the
line (this will give you the equivalent of predict_x and predict_y but
generalized to ND).  Sorry if this is already included, I haven't read the
code.